.github/workflows/update_iers.yml

```diff
@@ -0,0 +1,24 @@
+name: Auto-update IERS tables
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
```

Comment:
Is this daily? Does it need to be so frequent?

Reply:
Daily does seem overkill since really this is needed only to keep any release roughly up to date. Ideally, it "knows" when a release is imminent and only then triggers...  But otherwise, monthly is probably good enough.